libsoftwarecontainer/src/gateway/cgroups/cgroupsgateway.h

```diff
@@ -112,11 +112,24 @@ class JSonError : public CgroupsGatewayError
     }
 };
 
+class InvalidValueError : public CgroupsGatewayError
+{
```

Comment:
@tobsan @breakreturn Perhaps one of you has an opinion about this. It doesn't really make sense, as @kursatkobya points out, to have two different Errors, since they both indicate the same type of error (invalid input).